packages/component-test-utils-react/src/shallow.js

```diff
@@ -2,45 +2,61 @@ const React = require('react');
 const {getHtml} = require('./methods/html');
 const {shallowMock} = require('./shallow-mock');
 const {createDispatcher} = require('./dispatcher/');
+const {dispatchEvent} = require('./methods/dispatchEvent');
 
 const {
   ReactCurrentDispatcher
 } = React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
 
 class ShallowRender {
-  constructor() {
+  constructor(component, config = {}) {
+    if (!component) {
+      return;
```

Comment:
I think that the developer is essential. WYT of throwing an error with an indication on this?

Reply:
It's also a good practice in DDD: you validate rules before creating objects => this means that you always rely on object that have valid shapes

Reply:
It's a quick fix about a issue with mocking. This code will be detroyed in hell